crates/teloxide-core/CHANGELOG.md

```diff
@@ -19,10 +19,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `ChatId::as_user` ([#905][pr905])
 - Implement `PartialEq<ChatId> for UserId` and `PartialEq<UserId> for ChatId` ([#905][pr905])
 - `ChatId::{MIN, MAX}` ([#905][pr905])
+- Add missing Message::filter_* functions: `filter_game`, `filter_venue`, `filter_video`, `filter_video_note`, `filter_voice`, `filter_migration`, `filter_migration_from`, `filter_migration_to`, `filter_new_chat_title`, `filter_new_chat_photo`, `filter_delete_chat_photo`, `filter_group_chat_created`, `filter_supergroup_chat_created`, `filter_channel_chat_created`, `filter_message_auto_delete_timer_changed`, `filter_invoice`, `filter_successful_payment`, `filter_connected_website`, `filter_write_access_allowed`, `filter_passport_data`, `filter_proximity_alert_triggered`, `filter_forum_topic_created`, `filter_forum_topic_edited`, `filter_forum_topic_closed`, `filter_forum_topic_reopened`, `filter_general_forum_topic_hidden`, `filter_general_forum_topic_unhidden`, `filter_video_chat_scheduled`, `filter_video_chat_started`, `filter_video_chat_ended`, `filter_video_chat_participants_invited`, `filter_web_app_data` and getters: `message_auto_delete_timer_changed`, `write_access_allowed`, `forum_topic_created`, `forum_topic_edited`, `forum_topic_closed`, `forum_topic_reopened`, `general_forum_topic_hidden`, `general_forum_topic_unhidden`, `video_chat_scheduled`, `video_chat_started`, `video_chat_ended`, `web_app_data` ([#982][pr982])
```

Comment:
```suggestion
- Missing `Message::filter_*` functions ([#982][pr982]): 
  - `filter_game`
  - `filter_venue`
  - `filter_video`
  - `filter_video_note`
  - `filter_voice`
  - `filter_migration`
  - `filter_migration_from`
  - `filter_migration_to`
  - `filter_new_chat_title`
  - `filter_new_chat_photo`
  - `filter_delete_chat_photo`
  - `filter_group_chat_created`
  - `filter_supergroup_chat_created`
  - `filter_channel_chat_created`
  - `filter_message_auto_delete_timer_changed`
  - `filter_invoice`
  - `filter_successful_payment`
  - `filter_connected_website`
  - `filter_write_access_allowed`
  - `filter_passport_data`
  - `filter_proximity_alert_triggered`
  - `filter_forum_topic_created`
  - `filter_forum_topic_edited`
  - `filter_forum_topic_closed`
  - `filter_forum_topic_reopened`
  - `filter_general_forum_topic_hidden`
  - `filter_general_forum_topic_unhidden`
  - `filter_video_chat_scheduled`
  - `filter_video_chat_started`
  - `filter_video_chat_ended`
  - `filter_video_chat_participants_invited`
  - `filter_web_app_data`
- Missing `Message` getters ([#982][pr982]):
  - `message_auto_delete_timer_changed`
  - `write_access_allowed`
  - `forum_topic_created`
  - `forum_topic_edited`
  - `forum_topic_closed`
  - `forum_topic_reopened`
  - `general_forum_topic_hidden`
  - `general_forum_topic_unhidden`
  - `video_chat_scheduled`
  - `video_chat_started`
  - `video_chat_ended`
  - `web_app_data`
```
Also the part about `Message::filter_*` should be in the `teloxide` changelog, not `teloxide-core`.